composer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "x3m/germanphonetic",
```

Comment:
@deezaster should this be `deezaster` or `x3m`?

Reply:
x3m is ok!